### PR TITLE
[tests-only] Add webui tests for markdown editor

### PR DESCRIPTION
**Pull request by saw-jan**
## Description
Adds webui tests for markdown editor

## How Has This Been Tested?

- test environment: local

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [ ] Code changes
- [ ] Unit tests added
- [x] Acceptance tests added
- [ ] Documentation ticket raised: <link>

### CODE REVIEW

File: tests/acceptance/features/webUIMarkdownEditor/markdownFile.feature [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+
+  Scenario: preview content of the file
+    When the user opens file "simple.md" in the markdown editor webUI
+    Then the file "simple.md" should be displayed in the markdown editor webUI
+    And the preview panel should have the content "simple markdown file" in the WebUI
+
+  Scenario: preview content of the file while editing
+    Given the user has opened file "simple.md" in the markdown editor webUI
+    When the user inputs the content "updating the file with new content" in the markdown editor webUI
+    And the preview panel should have the content "updating the file with new content" in the WebUI
```
Comment (individual-it) [inline tests/acceptance/features/webUIMarkdownEditor/markdownFile.feature:53]: Then?

File: tests/acceptance/features/webUIMarkdownEditor/markdownFile.feature [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    Then the file should have content "updated content" in the markdown editor webUI
+
+  Scenario: append new content in a markdown file
+    Given the user has opened file "simple.md" in the markdown editor webUI
+    When the user appends the content " new content added" in the markdown editor webUI
+    And the user saves the file in the markdown editor webUI
+    And the user closes the markdown editor using the webUI
+    And the user opens file "simple.md" in the markdown editor webUI
+    Then the file "simple.md" should be displayed in the markdown editor webUI
+    Then the file should have content "simple markdown file new content added" in the markdown editor webUI
```
Comment (individual-it) [inline tests/acceptance/features/webUIMarkdownEditor/markdownFile.feature:34]:
```suggestion
    And the file should have content "simple markdown file new content added" in the markdown editor webUI
```

File: tests/acceptance/features/webUIMarkdownEditor/markdownFile.feature [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+  Scenario: close the markdown editor without saving the updated content
+    Given the user has opened file "simple.md" in the markdown editor webUI
+    When the user inputs the content "updated content" in the markdown editor webUI
+    And the user closes the markdown editor using the webUI
+    And the user opens file "simple.md" in the markdown editor webUI
+    Then the file "simple.md" should be displayed in the markdown editor webUI
+    Then the file should have content "simple markdown file" in the markdown editor webUI
+    Then the file should not have content "updated content" in the markdown editor webUI
```
Comment (individual-it) [inline tests/acceptance/features/webUIMarkdownEditor/markdownFile.feature:43]: And
And

**Review by individual-it — CHANGES_REQUESTED** [review]
After saving the file I would suggest to check the content via webdav API and not UI
we need a test that checks that the correct content is displayed on the UI, but checking that the content was correctly saved should happen by webdav, no need to re-open the file again

File: tests/acceptance/features/webUIMarkdownEditor/markdownFile.feature [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+    When the user appends the content " new content added" in the markdown editor webUI
+    And the user saves the file in the markdown editor webUI
+    And the user closes the markdown editor using the webUI
+    Then as "user1" the file "simple.md" should have the content "simple markdown file new content added"
+
+  Scenario: close the markdown editor without saving the updated content
+    Given the user has opened file "simple.md" in the markdown editor webUI
+    When the user inputs the content "updated content" in the markdown editor webUI
+    And the user closes the markdown editor using the webUI
+    Then as "user1" the file "simple.md" should not have the content "updated content"
```
Comment (individual-it) [inline tests/acceptance/features/webUIMarkdownEditor/markdownFile.feature:37]: no need to check that, the next step will check the correct content and that makes sure the content is not any of the other infinite possibilities

**Review by individual-it — CHANGES_REQUESTED** [review]

**Review by individual-it — APPROVED** [review]

**Review by phil-davis — APPROVED** [review]

**Pull request merged**